src/main/java/org/blockchain_innovation/factom/identiy/did/json/CustomServiceSerializer.java

```diff
@@ -4,11 +4,10 @@
 import com.google.gson.JsonObject;
 import com.google.gson.JsonSerializationContext;
 import com.google.gson.JsonSerializer;
+import com.google.gson.JsonSyntaxException;
```

Comment:
Only notice this now and probably was already the case. We need to split the Json/serialization into modules like in the Factom Java client. Right now we have deps to Gson even if the developer wants to use Yason/Jee for instance.

Doesn't need to happen now, but we need a TODO at least

Reply:
Good point. We also have a compile dependency on the `factom-did-models-java-gson`. Will that impede the use of other serializers/deserializers as the models have GSON annotations? (also, added the todo).

Reply:
Yes it will

Reply:
I guess for now we can just choose that you have to use GSON as the implementation for the identity client, but we have the pluggable JSON and logging support in the Factom client for a reason of course ;)

I always like it when something that is clearly a library/client has very minimal dependencies and/or SPI/plugin support.

Reply:
Ok, so I moved the dependency to `testCompile` and updated the readme, but it seems we don't have a generator in `factom-identity-schema` to generate any models other than through use of gson. I added a todo in the README as there should be a list of options for the models, but currently it's a list of size 1.